next-react-query-tailwind/src/components/RepoNavigation/index.ts

```diff
@@ -1 +0,0 @@
-export { default as default } from './RepoNavigation';
```

Comment:
If you delete this file, it will break all imports of `RepoNavigation` in other components. They assume an `index.ts` file inside the `RepoNavigation` directory with a default export, which is named `RepoNavigation` by the import itself.